### DISCUSSION TITLE
I'm not able to register as a Author on mailmap.py

### DISCUSSION THREAD

**Discussion by aayu-sha** [sympy/sympy#26581] [General]
I use Ubuntu 22.04 and I'm not able to register my name as a new contributor on the file mailmap.py

**Comment by sylee957** [comment]
If you refer by your changes in the PR https://github.com/sympy/sympy/pull/26580

I think that you have broken the `mailmap_check.py` script because of the change

https://github.com/sympy/sympy/blob/dd08407666b253cd78b684fc7dc2c3b78d58865b/bin/mailmap_check.py#L333

Please remove this line and try again.
